src/components/molecules/NavSearchBar/NavSearchBar.scss

```diff
@@ -18,58 +29,66 @@
   }
 }
 
-.nav__search input {
-  width: 18.125rem;
-  text-align: left;
-  background-color: $saturated-black;
-  border: solid 2px rgba(0, 0, 0, 0.13);
-  font-size: 0.9em;
-}
-
-.nav-search-results {
-  z-index: z(navbar-nav-search-results);
-
+.nav-dropdown {
   position: fixed;
-  top: 60px;
-
-  min-width: 18.125rem;
-
+  z-index: 1;
+  top: 0;
   height: auto;
-  max-height: 75vh;
-  overflow-y: auto;
-
-  padding: 20px;
-  box-shadow: 0 10px 30px 0 rgba(0, 0, 0, 0.5);
-  border-radius: 0 0 24px 24px;
-  transition: transform 400ms cubic-bezier(0.23, 1, 0.32, 1);
+  width: $search-results-width;
+  margin-top: 66px;
+  width: 290px;
   background-color: $saturated-black;
+  text-align: left;
+  box-shadow: 0 10px 30px 0 rgba($black, 0.5);
+  border-radius: 0 0 $border-radius--input $border-radius--input;
+  transform: translateY(-100px);
+  transform-origin: top right;
+  transition: all 400ms cubic-bezier(0.23, 1, 0.32, 1);
+  pointer-events: none;
 
-  .nav-search-result-number {
+  &.show {
+    transform: scale(1) translateY(0px);
+    opacity: 1;
+    pointer-events: all;
+  }
+
+  .nav-dropdown-title {
+    padding: 0 12px;
     opacity: 0.5;
     font-size: 0.9rem;
+    b {
+      font-weight: 500;
+    }
   }
 
-  .row {
-    padding: 10px;
-    flex-direction: row;
+  &.nav-dropdown--search {
+    width: 290px;
+    max-height: 480px;
+    border-radius: 0 0 $border-radius--input $border-radius--input;
+  }
+}
 
-    &:hover {
-      background-color: rgb(80, 74, 87);
-    }
+.nav-dropdown--search-results {
+  overflow-y: auto;
+  max-height: 385px;
 
-    .result-avatar {
-      width: 50px;
-      height: 50px;
-      border-radius: 5px;
-      margin-right: 10px;
-      background-position: center;
-      background-size: cover;
-    }
+  // body scrollbar
+  &::-webkit-scrollbar-track {
+    border-radius: 3px;
+    background-color: rgba(0, 0, 0, 0);
+  }
 
-    .result-info {
-      display: flex;
-      flex-direction: column;
-      justify-content: center;
-    }
+  &::-webkit-scrollbar {
+    width: 8px;
+    height: 8px;
+    background-color: rgba($black, 0);
+  }
+
+  &::-webkit-scrollbar-thumb {
+    border-radius: 4px;
+    background-color: rgba($white, 0.3);
+  }
+  &::-webkit-scrollbar-thumb:hover {
+    background-color: rgba($white, 0.5);
   }
```

Comment:
This scrollbar code seems to be repeated all over the place in the app. Can we extract it to use SCSS mixins/similar rather than being duplicated everywhere?

- https://sass-lang.com/documentation/at-rules/mixin

Reply:
created a scrollbar mixin: https://github.com/sparkletown/sparkle/pull/1328/commits/df4541b69ac6c3adc545fc92aa4d89f3df3e3736

Reply:
Awesome! :)